aviary/models/test_aircraft/aircraft_for_bench_GwFm.csv

```diff
@@ -163,7 +163,7 @@ aircraft:landing_gear:fixed_gear,True,unitless
 aircraft:landing_gear:main_gear_location,0.15,unitless
 aircraft:landing_gear:main_gear_mass_coefficient,0.85,unitless
 aircraft:landing_gear:main_gear_mass_scaler,1.1,unitless
-aircraft:landing_gear:main_gear_mass,6366.615,lbm
+#aircraft:landing_gear:main_gear_mass,6366.615,lbm
```

Comment:
I noticed GASP mass was never actually using this variable - it was using "main_gear_mass". I updated the GASP fixed mass to properly use Aircraft.Landing_Gear.MASS, however that is a calculated value so defining it here isn't necessary. No test values changed because of it (since we were setting it to the "correct" value here anyway).

Reply:
As an FYI defining this variable in the csv and calculating it actually causes an error, so perhaps there is also an issue with overrides?